styleguide/components/31-molecules/readspeaker-button/readspeaker-button.config.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  title: 'Readspeaker button',
+  status: 'ready',
+  handle: 'readspeaker-button',
+  context: {
+    text: 'Read more'
```

Comment:
Would hard-code this, it's not optional.
(also, this translation is not correct)